fatoptimizer/builtins.py

```diff
@@ -8,13 +8,13 @@
 
 
 def _chr_check_args(args):
-    code_point = args[0]
-    return 0 <= code_point <= 0x10ffff
+    #   args[0] => code_point
+    return 0 <= args[0] <= 0x10ffff
```

Comment:
I don't understand the purpose of this change. It makes the code *less* readable. The fact that you added a comment means that the code is unclear. I consider that if the code is clear enough, it doesn't require a comment.

Your commit message says "Elimination of unused variables", but it's wrong: the variable *is* used.